### PR TITLE
Fixed blog post and front page blog links

### PR DESCRIPTION
**Pull request by rehrar**

### CODE REVIEW

File: _posts/2018-03-01-core-team-announcement.md [hunk 1]
```diff
@@ -11,18 +11,18 @@ The year 2018 is well underway, with many exciting new technologies on the horiz
...
 
 The responsibilities of the Core Team are as follows (in no particular order):
-    - Act as primary trusted arbiters of the Forum Funding System on behalf of the community, so as to ensure the completion of all projects to the satisfaction of the community.
-    - Manage the codebase of the Monero Project, which includes merging code on Github, keeping backups, and ensuring the safety, security, and free access of the code from any party.
-    - Steward the general donation fund, and spending the Monero there on anything they see fit to further the Monero Project.
-    - Act as trusted signers and distributors of reference clients for the Monero coin, and other related technologies.
-    - Set a direction and vision for the Monero Project
+- Act as primary trusted arbiters of the Forum Funding System on behalf of the community, so as to ensure the completion of all projects to the satisfaction of the community.
```
Comment (leonklingele) [inline _posts/2018-03-01-core-team-announcement.md]: How is this related in any way and went through code review?
@luigi1111 @rehrar

File: _i18n/en/index.md [hunk 2]
```diff
@@ -118,7 +118,7 @@
                               {% for post in site.posts limit:2 %}
                               <div class="row start-xs">
                                                     <div class="col">
-                                                        <p><a href="{{site.baseurl}}/{{ post.url }}">{{ post.title }}</a></p>
+                                                        <p><a href="{{site.baseurl}}{{ post.url }}">{{ post.title }}</a></p>
```
Comment (leonklingele) [inline _i18n/en/index.md]: Also, why did this drop `site.baseurl`?

File: _posts/2018-03-01-core-team-announcement.md (hunk 1 above)
Comment (mattcode55) [reply _posts/2018-03-01-core-team-announcement.md]: What's the issue?

Comment (leonklingele) [reply _posts/2018-03-01-core-team-announcement.md]: Leading whitespaces were removed, should have been done in a different PR.
